modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -231,11 +255,28 @@ class cv::gimpl::GExecutor::Input final: public cv::gimpl::GIslandExecutable::II
 class cv::gimpl::GExecutor::Output final: public cv::gimpl::GIslandExecutable::IOutput
 {
     cv::gimpl::Mag &mag;
-    virtual GRunArgP get(int idx) override { return magazine::getObjPtrExec(mag, desc()[idx]); }
-    virtual void post(GRunArgP&&) override { } // Do nothing here
-    virtual void post(EndOfStream&&) override {} // Do nothing here too
+    std::unordered_map<const void*, int> out_idx;
```

Comment:
Is this ok to use `void*` here? Maybe it's better to utilize `any`

Reply:
I might be completely wrong but haven't seen `void*` before in backends/executor

Reply:
`any` isn't better in any form, as `void*` is used as some unique identifier. It's underlying value is never accessed.